init.lua

```diff
@@ -6,6 +6,10 @@ local chat3_exists = minetest.get_modpath("chat3")
 local registered   = {}
 local default
 
+-- Load mod storage
+
```

Comment:
Alright so I'm being extremely, probably unnecessarily and maybe even a tiny bit obnoxiously picky right now, but this whitespace has really been annoying me :joy:

Reply:
xD I'll remove it!